src/braket/aws/aws_device.py

```diff
@@ -68,7 +68,8 @@ def __init__(self, arn: str, aws_session: Optional[AwsSession] = None):
             the QPU is located in.
 
             See `braket.aws.aws_device.AwsDevice.DEVICE_REGIONS` for the AWS Regions provider
-            devices are located in.
+            devices are located in across the AWS Braket service.
+            This is not a device specific tuple.
```

Comment:
```suggestion
            Note:
                This is not a device specific tuple.
```

Reply:
This section already is a note - this would make it nested.